src/core/reactor.cc

```diff
@@ -1172,7 +1172,10 @@ cpu_stall_detector_posix_timer::cpu_stall_detector_posix_timer(cpu_stall_detecto
     struct sigevent sev = {};
     sev.sigev_notify = SIGEV_THREAD_ID;
     sev.sigev_signo = signal_number();
-    sev._sigev_un._tid = syscall(SYS_gettid);
+#ifndef sigev_notify_thread_id
```

Comment:
could you please use the imperative mood in the title, and put the explanations in the body of the commit message? the title could look like :

> reactor: access sigevent.sigev_notify_thread_id with a macro

and might want to note that the name macro of `sigev_notify_thread_id` comes from musl libc, which defines a macro with the same name for accessing the corresponding field in `sigevent`.

Reply:
okay will modify description

Reply:
this comment is not addressed yet. could you please reconsider it?